.idea/gradle.xml

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey, @ajaygalagali why you deleted the whole file, we don't want any change in this file so just leave it as it is instead of deleting the whole file

Reply:
Updated